Makefile

```diff
@@ -99,3 +99,14 @@ build-image-collector:
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_IMAGE_TAG} \
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_VERSION} \
 		-f Dockerfile .
+
+build-and-deploy-image-collector:
+	docker build -f Dockerfile -t collector-image
+	docker tag collector-image quay.io/testnetworkfunction/collector
+	docker push quay.io/testnetworkfunction/collector
+	oc apply -f /home/shmoran/go/src/github.com/test-network-function/collector-deployment/k8s/collector-deployment.yml
```

Comment:
This is hardcoded to your local system but I get what you're trying to do.  Maybe we should (temporarily) clone down the collector-deployment repo here and then use `sed` to replace the image with the newly tagged and pushed dev build?

Reply:
Yes the local URL is a mistake, my changes didn't save. I'll change it

Reply:
The main reason for this make path as I see it, is for testing purposes when working on the collector application\deployment. So maybe using:
`oc apply -f ./k8s/collector-deployment.yml` 
and changing the collector deployment to use `{REGISTRY}/${COLLECTOR_IMAGE_NAME}:dev` image instead of `latest` tag will do the work? what do you think?

Reply:
There is no different between using latest or dev tag. Use some version. And find a way to increment the version in each build.

Reply:
Though not every push i a new version, usually are to check different test cases. so I think we might not want to save those build aren't we?

Reply:
So isn't same as using latest?

Reply:
I would also agree to use any tag for being "latest" discouraged to use.

Reply:
No, image not tagged gets latest tag by the registry. If you push multiple images with the same name and no tag, you get the most recent one as latest.